cryptofeed/config.py

```diff
@@ -25,9 +26,11 @@ def __setitem__(self, key, value):
         super().__setitem__(key, value)
 
     def __getattr__(self, item):
+        # print(f'Config: found {item!r}', file=sys.stderr, flush=True)
```

Comment:
did you mean to leave these commented out?

Reply:
i'd think we should just remove them

Reply:
Done :+1: